src/ahocorasick.pyi

```diff
@@ -1,9 +1,8 @@
-from typing import TypeVar, Generic, Iterator, Tuple
-
+from typing import Generic, Iterator, TypeVar
 
 T = TypeVar("T")
 
 class Automaton(Generic[T]):
     def add_word(self, word: str, value: T) -> bool: ...
     def make_automaton(self) -> bool: ...
-    def iter(self, str) -> Iterator[Tuple[str, T]]: ...
+    def iter(self, str) -> Iterator[tuple[str, T]]: ...
```

Comment:
Is this python 3.8 compatible?

Reply:
This stub file is only used by mypy and not at runtime. `ruff` will also happily change the syntax even when running with `target-version = py38`.